web-admin/vite.config.ts

```diff
@@ -2,6 +2,13 @@ import { sveltekit } from "@sveltejs/kit/vite";
 import type { UserConfig } from "vite";
 
 const config: UserConfig = {
+  resolve: {
+    alias: {
+      "@rilldata/web-admin": "/src",
+      "@rilldata/web-common": "/../web-common/src",
+      "@rilldata/web-local": "/../web-local/src",
```

Comment:
Are these supposed to start with a `/`? Looks like a root path

Reply:
For whatever reason it breaks without the forward slash. [Docs here](https://vitejs.dev/config/shared-options.html#resolve-alias). It's resolving from the Vite project root, which SvelteKit sets.